src/nodejs/lambda-layers/message-util/src/message-util.js

```diff
@@ -49,28 +125,31 @@ async function getSecretsValues() {
 async function send(user, eventMessage, customTemplateFunction, ses) {
   try {
     const bodyArray = await createEmailHtml({ user, eventMessage, customTemplateFunction });
-    const payload = {
-      Source: sourceEmail,
-      Destination: {
-        ToAddresses: [user.email]
-      },
-      Message: {
-        Subject: {
-          Data: 'EDPUB Notification'
-        },
-        Body: {
-          Text: {
-            Data: bodyArray[0],
-            Charset: 'UTF-8'
-          },
-          Html: {
-            Data: bodyArray[1],
-            Charset: 'UTF-8'
-          }
+    const nasaLogo = await getAttachmentAsBase64String({
+      bucket: process.env.DASHBOARD_BUCKET,
+      key: 'images/app/src/assets/images/nasa-logo.d7dbc5e408ccd79bb7578f3358413d69.png'
+    });
+
+    const rawEmail = getRawFromTemplate({
+      subject: 'EDPUB Notification',
+      from: sourceEmail,
+      to: user.email,
+      htmlText: bodyArray[1],
+      plainText: bodyArray[0],
+      images: [
+      // { data: imageAttachment, name: 'image1.jpg' } // attachment code will be added later
```

Comment:
Thanks for adding this here; I think it will help a lot with future integrations!